### DISCUSSION TITLE
Disable auto-hide toolbar when mouse not moving

### DISCUSSION THREAD

**Discussion by oussamabonnor1** [iTwin/itwinjs-core#4397] [Q&A] [answered]
There's a mouse move listeners that shows/hides the toolbar widget automatically. Can it be disabled ?

If it helps, the class being used to achieve this is `.nz-hidden`. I rather not hardcode CSS to override this if possible.

**Comment by NancyMcCallB** [comment, marked as answer]
Yes, this behavior is optional.

Import UiShowHideManager from appui-react and simple call  UiShowHideManager.autoHideUi = false in your startup code.

Nancy

**Reply by oussamabonnor1** [reply]
That works, thanks!
